app/controllers/people_roles_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PeopleRolesController < CrudController
+  self.permitted_attrs = %i[level percent role_id person_id]
```

Comment:
Wenn du %i brauchst kannst du mit runden Klammern arbeiten

Reply:
Ich ändere die eckigen Klammern nicht, da diese in jedem Controller so verwendet werden.